corehq/apps/app_manager/templates/app_manager/partials/appnav_menu_header.html

```diff
@@ -4,7 +4,12 @@
   <div class="appnav-name-settings">
     <a href="{% url "app_settings" domain app_id %}"
        class="appnav-responsive{% if settings_active %} active{% endif %}">
-      <i class="fa-solid fa-gear"></i>
+      <i
+        class="fa-solid fa-gear"
+        title="{% trans 'Application Settings' %}"
```

Comment:
```suggestion
        title="{% trans_html_attr 'Application Settings' %}"
```

Reply:
(handles both translation and quotes)

Reply:
I tried this locally and noticed that `trans_html_attr` does not create a translation entry for the string. I'd imagine this would work if "Application Settings" was translated elsewhere, but otherwise translators will never have an opportunity to provide the correct translation. Leaving it as is for now

Reply:
yeah, while looking at that function for this PR I realized it's fundamentally flawed. Looking into it.

Reply:
Hmm, that was my initial impression, but then I convinced myself it was fine (until I realized it didn't register the translation string). Curious where the flaw is. I ended up looking at the django source code, and tbh, I'm not sure how django gets `makemessages` to detect their `translate` tags -- it looked like everything mapped back to `gettext`, which should be what is happening with `trans_html_attr`.

Reply:
I believe `makemessages` does some kind of static analysis. My understanding is primarily based on [our translations docs](https://commcare-hq.readthedocs.io/translations.html). @esoergel and I chatted about this a bit and came to the same conclusion you did, that your code will only translate if "Application Settings" is translated elsewhere in HQ.

https://github.com/dimagi/commcare-hq/pull/36153 updates `html_attr` so that you can use `{% html_attr _("Application Settings") %}`